axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -941,6 +953,12 @@ InvoiceSet.length > 0))"
 		<action name="action-invoice-method-fill-in-lines"/>
 		<action name="action-invoice-method-fill-advance-payment-set"/>
 	</action-group>
+	
+	    <action-group name="action-invoice-group-payment-condition-onchange">
```

Comment:
please change the name of this action-group, it can be called when other fields are changed (like invoiceDate) so the name can be misleading.